### ISSUE TITLE
Feature Suggestion: Simple Demo Drone Programs

### ISSUE THREAD

**Issue by nirakoji** [TeamPneumatic/pnc-repressurized#841]
Whether in the PNCR book or in a separate book, it would be useful to have a selection of simple demo programs to learn from.

I have basic programming knowledge, so learning from other programs and reverse-engineering the logic/flow is a useful part of the process for me. I imagine it might be helpful for others who are in similar positions to have a feature like this baked into the mod.

Suggestions for demo programs:
- The Basic Drones (Collector, Logistics, Harvesting, Guard) as done via Programmable Drone.
- A basic "player/entity pick up and drop" program

And beyond that, I'm not even sure what else I'd need to know to get a good handle on drone programming. ;)

**Comment by HipHopHuman** [comment]
Definitely an Autocrafting program, complete with "Input Chest" / "Output Chest". A lot of the people I brag about my PNC builds to are always like "Wow, I didn't know PNC drones could autocraft" and I remember trying to set up my first autocrafting widget and having a bit of trouble with it.

**Comment by desht** [comment]
Obviously, programs are much too large to place in the manual.  I'm open to example of programs being uploaded to pastebin or similar (pastebin is best since the GUI directly supports it), but I don't personally have time to create any such examples right now.

So: write some programs, upload them to pastebin, and add links and descriptions (long enough to be instructive, short enough to fit a small number of manual pages - include the number of puzzle pieces required too) to this issue.  And I can look at adding a section to the manual for example programs...
